### DISCUSSION TITLE
Problem with new project

### DISCUSSION THREAD

**Discussion by gaidygomez** [adonisjs/core#3129] [General]
Hi, guys.

I had tried create a new project with adonisjs folliwing the guideline in documentation and although everything seems normal but when i try create a new model with comman `node ace make:model User -m`, first terminal say me that flag -m is unknown, then delete -m flag and execute again the same command then terminal say me `make:model` is command not found. This is awesome for me.

**Comment by thetutlage** [comment]
Have you followed the database setup guide? https://docs.adonisjs.com/guides/database/introduction

**Reply by gaidygomez** [reply]
Yeah...  was a configuration problem on the part of the project leader. Sorry

**Reply by finestgecko** [reply]
@gaidygomez  I'm experiencing a [similar problem](https://github.com/adonisjs/core/discussions/3132) (except in my case it's `node ace serve --watch` that's saying `E_INVALID_FLAG: Unknown flag "watch"`). Could you share how you fixed it?
